core/main/src/bootstrap/start_fbgateway_step.rs

```diff
@@ -108,6 +110,10 @@ impl Bootstep<BootstrapState> for FireboltGatewayStep {
             return Err(RippleError::BootstrapError);
         }
         TelemetryBuilder::send_ripple_telemetry(&state.platform_state);
+        info!(
+            "Ripple Total Bootstrap time: {}",
+            Instant::now().duration_since(state.start_time).as_millis()
```

Comment:
I assume that `info!()` log will be removed from production build. There is some performance impact on using `now() `. It is advised to use this function `sparingly`.

Reply:
This log is once per Ripple successful bootstrap.